src/ocp_postprocess/ip_rename.rs

```diff
@@ -181,31 +184,22 @@ async fn extract_original_dual_stack_ips(etcd_client: &Arc<InMemoryK8sEtcd>) ->
 }
 
 async fn fix_etcd_resources_dual_stack(etcd_client: &Arc<InMemoryK8sEtcd>, original_ips: &[String], new_ips: &[String]) -> Result<()> {
-    let original_ipv4 = &original_ips[0];
-    let original_ipv6 = &original_ips[1];
-
-    let new_ipv4 = &new_ips[0];
-    let new_ipv6 = new_ips.get(1).context("Second IP (IPv6) is required for dual-stack processing")?;
-
-    log::info!(
-        "Applying dual-stack IP changes - IPv4: {} → {}, IPv6: {} → {}",
-        original_ipv4,
-        new_ipv4,
-        original_ipv6,
-        new_ipv6
+    ensure!(!original_ips.is_empty(), "at least one original IP is required");
```

Comment:
Any reason this ensure is present in this function and not also in the other one?